templates/swift/Sources/OAuth/WebAuthComponent.swift.twig

```diff
@@ -103,6 +111,15 @@ public class WebAuthComponent {
         )
     }
 
+    private static func validateRequiredCookieParts(_ cookieParts: [String: String]) -> (domain: String, key: String, secret: String)? {
+        guard let domain = cookieParts["domain"],
+                let key = cookieParts["key"],
+                let secret = cookieParts["secret"] else {
+            return nil
+        }
+        return (domain, key, secret)
+    }
+
```

Comment:
Let's do this inline instead of a separate function

Reply:
sound good.